11/umbraco-cms/SUMMARY.md

```diff
@@ -14,7 +14,7 @@
     * [Install using .NET CLI](fundamentals/setup/install/install-umbraco-with-templates.md)
     * [Install using Visual Studio](fundamentals/setup/install/visual-studio.md)
     * [Local IIS With Umbraco](fundamentals/setup/install/iis.md)
-    * [Install Umbraco With Visual Studio Code](fundamentals/setup/install/install-umbraco-with-vs-code.md)
+    * [Install using Visual Studio Code](fundamentals/setup/install/install-umbraco-with-vs-code.md)
```

Comment:
You need to also update the title of the article, when updating the link to it in the navigation 💪 
They are tied together, so if you don't do it, GitBook will change this back to use the article title again 😅